pkg/config/config.go

```diff
@@ -132,6 +132,9 @@ func NewNSXOperatorConfigFromFile() (*NSXOperatorConfig, error) {
 	if err != nil {
 		return nil, err
 	}
+	if !cfg.Section("ha").HasKey("enable") {
+		nsxOperatorConfig.HAConfig.EnableHA = true
+	}
```

Comment:
Do you want to use pointer for this?
```
type HAConfig struct {
	EnableHA *bool `ini:"enable"`
}
```

Reply:
WCP forgets to set ha.enable but set replica 2, it would be confusing, so set EnableHA default true to avoid this undefined behaviour.

Please remove WCP in comments, it's internal words not for public.

Reply:
Now is

Reply:
> WCP forgets to set ha.enable but set replica 2, it would be confusing, so set EnableHA default true to avoid this undefined behaviour.
> 
> Please remove WCP in comments, it's internal words not for public.

OK

Reply:
> If user forgets to set ha.enable but set replica 2, it would be confusing, so set EnableHA default true to avoid this undefined behavior.

We can say  "NCP ha.enable default value is true, so setting EnableHA default true as well to align with NCP default setting.

Reply:
> Now is
> 
> type HAConfig struct { EnableHA bool `ini:"enable"` }
> 
> Like others in this file.

In golang, if you want to check if an option is unset, we should use pointer in the target struct, then this parameter will be `nil` when it is not set or invalid.

Reply:
Thanks, it is more Golang taste in this way, updated.

Reply:
> > If user forgets to set ha.enable but set replica 2, it would be confusing, so set EnableHA default true to avoid this undefined behavior.
> 
> We can say "NCP ha.enable default value is true, so setting EnableHA default true as well to align with NCP default setting.

Appended.